README.md

```diff
@@ -1,48 +1,248 @@
+![image](https://github.com/user-attachments/assets/d804ab89-2d21-467a-b80e-7bf3082ea614)
+
+Login to the application using the credentials created during the installation process to access the MESA dashboard.
+
+![image](https://github.com/user-attachments/assets/e5a0d4f8-2dc7-4448-99e6-788c0d08e8b6)
+
+Configuring Scan Settings
+--------------
+Prior to running any scans, browse to the settings tab to configure the scan scope and pertinent deliverable variables.
+
+![image](https://github.com/user-attachments/assets/3b13e3bd-ab9e-4fae-b5e4-3d2e28b5af61)
+
+Update the following variables in the settings tab:
+
+| Variable | Description | Default | Required |
+|----------|-------------|---------|---------|
+| Project Name | The name used to track the project | RV#### | yes |
+| Customer Name | The customer full name as it will appear on the deliverables | Acme Corporation | yes |
+| Customer Initials | The customer shortname as it will appear on the deliverables | ACME | yes |
+| In-Scope Systems | The individual IP addresses or cidr blocks to be included in scanning activity | none | yes |
+| Excluded Systems | The individual IP addresses or cidr blocks to be excluded from scanning activity | none | no |
+
+MESA Job Definitions
+--------------
+Once the settings have been configured browse to the dashboard tab to begin running the scans.
+
+![image](https://github.com/user-attachments/assets/1091208a-12a2-463e-81aa-960d47dc9497)
+
+| Job Title | Description | Required | 
+|----------|-------------|---------|
+| Host Discovery Scans | Performs an nmap discovery scan against the provided scope. | yes |
+| SMB Signing Checks | Performs a netexec scan to identify windows systems lacking smb signing within the provided scope. | no |
+| Web Application Enumeration | Performs an aqutone web application enumeration scan against the provided scope. | no |
+| Encryption Check | Performs a discovery scan to identify the exisitence of non encrypted protocols within the provided scope. Performs a secondary ssl scan against discovered web applications to check for deprecated ciphers. | no |
+| Default Logins Check | Performs automated checks using nuclei for default login information against web applications and network devices within the provided scope. | no |
+| Vulnerability Scan | Performs vulnerability scans using nuclei to identify critical, high, and medium findings. | no |
+| Full Port Scan | Performs an nmap full port scan against the provided scope. | no |
+
+Running MESA Jobs
+--------------
+
+MESA jobs can be ran individually or automatically in sequential order. To run the scans individually simply select the ![image](https://github.com/user-attachments/assets/208c9b72-7468-4421-a2f0-f9c7434b352f) button for the job you would like to run. 
+
+>[!IMPORTANT]
+>It is recommended to run the `Host Discovery Scans` job prior to running any other scans. This will ensure that a `live hosts` file is generated, expediting the time spent performing scans.
```

Comment:
recommended or required? Kind of tying this in with the comment above about what required means

Reply:
Added another note to explain why the scan is required and modified `recommended` to `required`.